stacker/commands/base.py

```diff
@@ -66,3 +67,17 @@ def run(self, options, **kwargs):
 
     def configure(self, options, **kwargs):
         pass
+
+    def get_context_kwargs(self, options, **kwargs):
```

Comment:
Maybe name this `additional_context`?  No strong opinion here, but it took me a moment to understand what you were doing.